Mijhail/Task1/secuencia.rb

```diff
@@ -0,0 +1,15 @@
+  def primtSecuence
+    secuences= Array.new
```

Comment:
Try to use a space 
sequences = Array.new 
Don't forget the principle of Readable